client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -94,6 +98,43 @@ const trackTaskDisplay = ( dispatch, task, siteId ) => {
 	);
 };
 
+const siteSetupListReducer = ( state, action ) => {
```

Comment:
Personally the way it was initially coded served me better. It seems that we add a lot of boilerplate code that doesn't enhance maintainability and also makes the code more error prone.

Reply:
I agree with this. On 2nd thoughts I will revert the changes to local state back to how it was.

Reply:
Reverted.